scripts/loop.bash

```diff
@@ -250,6 +250,15 @@ cancel_build_num(){
 }
 
 
+get_wait_time() {
+    local current_time
+    current_time=$(date +%s)
+    echo "$((current_time - start_time))"
```

Comment:
😆  this broke since change #139  meant it returned a timestamped string not integer